webscraper.py

```diff
@@ -20,7 +20,7 @@ def lookup_ability(self, args):
 	abiDict = self.server.db.get_cached_ability(name)
 	if abiDict is not None:
 		logger.log("lookup_ability", "cache hit")
-		return self.respond({"success":True, "value":abiDict["contents"]})
+		return self.respond({"success":True, "value":'\n'.join(abiDict["contents"])})

```

Comment:
contents is an array. This just makes the response consistent.

Reply:
TODO: Now that I'm looking at this, we should make it match the add ability logic where we get approximations first since that should be relatively fast. Or check cache twice?